### ISSUE TITLE
[v7] Wrong queue for responsive image generation

### ISSUE THREAD

**Issue by Omranic** [spatie/laravel-medialibrary#888]
__**ISSUE UPDATED!**__

It seems that original responsive image generation is queued on the default Laravel queue, not the custom queue specified in medialibrary config. Looks like conversions are sent to the custom queue, but responsive images generation for the original image are sent to the default laravel queue.

Even if we've `withResponsiveImages` on both the original image, and on conversions, it send two queue messages to two different queues at the same time, one for the **correct** custom queue for conversions, and another for the original responsive image on the default laravel queue, strange!

Controller code:
```php
$model->addMediaFromRequest('file')
     ->withResponsiveImages()
     ->toMediaCollection();
```

Model code:
```php
    public function registerMediaConversions(Media $media = null)
    {
        $this->addMediaConversion('thumb')->withResponsiveImages()->greyscale();
    }
```

**Comment by freekmurze** [comment]
I'll take a look soon, the desired behaviour is that:

- the queue specified in the config file is used
- responsive images generated for the original image are always queued
- responsive images generated for a conversion are performed in the same job as the conversions themselves. If the conversion is not queued,  the responsive image generation shouldn't be queued.

I welcome a PR with tests that correct bugs in the code towards this desired behaviour.

**Comment by Omranic** [comment]
Fix PR submitted #889
